testdata/redundant-import-alias.go

```diff
@@ -1,12 +1,24 @@
 package fixtures
 
-import(
+import (
+	runpb "cloud.google.com/go/run/apiv2/runpb" // MATCH /Import alias "runpb" is redundant/
+
+	md5 "crypto/md5" // MATCH /Import alias "md5" is redundant/
+
+	strings "strings" // MATCH /Import alias "strings" is redundant/
+
 	"crypto/md5"
-		"strings"
-		_ "crypto/md5"
-		str "strings"
-		strings "strings"  // MATCH /Import alias "strings" is redundant/
-		crypto "crypto/md5"
-		md5 "crypto/md5"  // MATCH /Import alias "md5" is redundant/
+	_ "crypto/md5" // MATCH /Import alias "_" is redundant/
```

Comment:
This is wrong. `_` is a special alias type for side effect loads. If your intention was to avoid duplicate imports, then this is a wrong rule to apply it here.

Reply:
I know this is wrong, but it is testdata file, in normal apps not used :)

Reply:
ok removed wrong example :)